examples/prediction/xgboost-high-level-apis.py

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Is there a specific reason to add this file instead of adding a private docker section into the notebook. If there is, please name the file appropriately. This can be confusing to users who may be expecting a notebook which is the primary way of invoking fairing.

Reply:
@abhi-g yes there was no real need for separate python example file.Have added prvate registry block in existing notebook example.